app/test_main.py

```diff
@@ -1 +1,49 @@
-# write your code here
+from unittest.mock import patch, MagicMock
+
+from app.main import can_access_google_page
+
+
```

Comment:
mb we can try to use parametrize or etc.?)
then code will be DRY imho

Reply:
good suggestion, tried to rewrite it with parametrized test and mock functions with defined return value to make test easier to read